src/User/Controller/CreateUserController.php

```diff
@@ -59,12 +59,12 @@ public function __construct(
     #[Post(
         path: self::API_PATH . '/user/',
         operationId: 'user_create',
-        summary: 'Create a new user.',
+        summary: 'create_user_summary',
```

Comment:
can you please change the translation keys so they match the operationIDs? 
e.g here `user_create_summary`

the idea would be that all of elements (assets, data objects, users ....) are then together when we sort the translations

Reply:
thx... missed that one after the merge :)

Done.